src/layouts/sidebar-sidecar/index.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import BaseNewLayout from 'layouts/base-new'
+import Footer from 'components/footer'
+import BreadcrumbBar, { BreadcrumbLink } from 'components/breadcrumb-bar'
+import EditOnGithubLink from 'components/edit-on-github-link'
+import Sidebar, { MenuItem } from 'components/sidebar'
+import Sidecar from 'components/sidecar'
+import { SidecarHeading } from 'components/sidecar/types'
+import s from './sidebar-sidecar-layout.module.css'
+
+interface DocsLayoutProps {
```

Comment:
Do we want to rename this too?
```suggestion
interface SidebarSidecarLayoutProps {
```

Reply:
What the heck, I thought I did a replace all on this! 😆